contracts/src/systems/cc_map/interface.cairo

```diff
@@ -0,0 +1,41 @@
+// ---------------------------------------- interface -------------------------------------
+#[starknet::interface]
+trait CCInterface<TContractState> {
```

Comment:
this should be exposed in the CC repo and imported, it shouldn't exist within eternum repo